libs/clients/national-registry/v3/src/lib/apiConfig.ts

```diff
@@ -1,16 +1,11 @@
 import { createEnhancedFetch } from '@island.is/clients/middlewares'
-import {
-  ConfigType,
-  LazyDuringDevScope,
-  XRoadConfig,
-} from '@island.is/nest/config'
+import { ConfigType, XRoadConfig } from '@island.is/nest/config'
 
 import { Configuration } from '../../gen/fetch'
 import { NationalRegistryV3ClientConfig } from './nationalRegistryV3.config'
 
 export const ApiConfig = {
   provide: 'NationalRegistryV3ClientProviderConfiguration',
-  scope: LazyDuringDevScope,
```

Comment:
We need this so it can run codegen in CI without environment variables present.

Reply:
Oh snap. Then we have to look into this. The notficationWorker.service does not run when this is defined, at least not in development. When Scope.REQUEST is present it wont run